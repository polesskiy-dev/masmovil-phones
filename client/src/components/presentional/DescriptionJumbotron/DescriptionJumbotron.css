.app-jumbotron {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;

    padding: 96px 32px;
}