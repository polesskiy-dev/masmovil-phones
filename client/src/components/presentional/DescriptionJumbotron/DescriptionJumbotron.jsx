import React from 'react';
import { FormattedMessage } from 'react-intl';

import './DescriptionJumbotron.css';

const DescriptionJumbotron = () => (
  <section className="app-jumbotron">
    <header>
      <h1 className="jumbotron-heading">
        <FormattedMessage id="DescriptionJumbotron.header" />
      </h1>
    </header>
    <p className="lead text-muted">
      <FormattedMessage id="DescriptionJumbotron.message" />
    </p>
    <a
      href="https://github.com/polesskiy-dev/masmovil-phones"
      target="_blank"
      rel="noopener noreferrer"
      className="btn btn-primary"
    >
      <FormattedMessage id="DescriptionJumbotron.repoLinkCaption" />
    </a>
  </section>
);

export default DescriptionJumbotron;
