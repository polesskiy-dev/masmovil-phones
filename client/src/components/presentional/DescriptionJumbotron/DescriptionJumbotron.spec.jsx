import React from 'react';
import renderer from 'react-test-renderer';
import _ from 'lodash/fp';

import withDefaultIntl from '../../../helpers/with-default-intl';
import DescriptionJumbotron from './DescriptionJumbotron';

it('<DescriptionJumbotron/ > renders correctly with all required fields', () => {
  const tree = renderer
    .create(withDefaultIntl(<DescriptionJumbotron onClick={_.noop} />))
    .toJSON();
  expect(tree).toMatchSnapshot();
});
