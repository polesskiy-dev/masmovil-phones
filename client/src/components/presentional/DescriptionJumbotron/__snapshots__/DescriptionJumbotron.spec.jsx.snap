// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DescriptionJumbotron/ > renders correctly with all required fields 1`] = `
<section
  className="app-jumbotron"
>
  <header>
    <h1
      className="jumbotron-heading"
    >
      <span>
        Phone Catalog App example
      </span>
    </h1>
  </header>
  <p
    className="lead text-muted"
  >
    <span>
      Very simple product catalog app with some modern React features
    </span>
  </p>
  <a
    className="btn btn-primary"
    href="https://github.com/polesskiy-dev/masmovil-phones"
    rel="noopener noreferrer"
    target="_blank"
  >
    <span>
      See source & description in Github
    </span>
  </a>
</section>
`;
